CHANGELOG.md

```diff
@@ -2,6 +2,7 @@
 
 ## Unversioned
 
+- Minor: Added badge highlighting. (#1704)
```

Comment:
This should be more descriptive and have Major changelog entry instead of minor. It's definietly one of the bigger things so far since v2.3.0

Reply:
I have updated this, please re-review.

Reply:
I guess what you wrote is fine, good stuff.